web/src/components/storage/LvmPage.tsx

```diff
@@ -57,14 +57,18 @@ import { _ } from "~/i18n";
  * that are already there.
  */
 function useLvmTargetDevices(): StorageDevice[] {
-  const availableDevices = useAvailableDevices();
+  const candidateDevices = useCandidateDevices();
   const systemDevices = useDevices("system", { suspense: true });
   const model = useModel({ suspense: true });
 
   const targetDevices = useMemo(() => {
-    const raids = (model.mdRaids || []).map((r) => systemDevices.find((d) => d.name === r.name));
-    return [...availableDevices, ...raids];
-  }, [availableDevices, systemDevices, model]);
+    const sids = candidateDevices.map((d) => d.sid);
+    const raids = (model.mdRaids || [])
```

Comment:
`Model.mdRaids` cannot be null.

Reply:
Fixed. Although this is temporary code that I expect to die in less than a month.